packages/react/src/definitions/kyc.ts

```diff
@@ -185,6 +185,30 @@ export interface KycLegalEntityData {
   legalEntity: LegalEntity;
 }
 
+export enum DocumentTypes {
+  IDCARD = 'ID Card',
+  PASSPORT = 'Passport',
+  DRIVERS_LICENSE = "Driver's License",
+  RESIDENCE_PERMIT = 'Residence Permit',
```

Comment:
Use the existing enum string values
![image](https://github.com/user-attachments/assets/8ce25473-ce6d-4c52-ad34-94efc693169d)